packages/dx-chart-core/src/plugins/zoom-and-pan/computeds.ts

```diff
@@ -200,6 +200,9 @@ export const getRect = (
   };
 };
 
-const checkInteraction = (interaction: Interaction, type: Interaction) => (
-  interaction === 'both' || interaction === type
-);
+const checkInteraction = (interaction: Interaction, type: Interaction) =>
+interaction === 'both' || interaction === type;
+
+/** @internal */
+export const setCursorType = (node: any, type: string) =>
+node.style.cursor = type;
```

Comment:
Why we return a value here? This function is not pure, I think we should return `void`.